media-video/pipewire/pipewire-0.3.24.ebuild

```diff
@@ -0,0 +1,228 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson eapi8-dosym
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/PipeWire/pipewire.git"
```

Comment:
@a17r , @thesamesam , this (and the one below, needing an ACK) are the only outstanding issues right now that anyone has brought up - should the ebuild keep using the GitHub mirror (controlled by the upstream, I think) or their [main GitLab instance](https://gitlab.freedesktop.org/pipewire/pipewire). And if the latter, should we prefer to download the tar.gz or the tar.bz2? I did google for an answer but so far found not even a discussion, much less guidance.

Reply:
I'd prefer the main GitLab instance and I'd say your choice of tar.gz, tar.bz2, although I always end up picking .gz out of habit (probably should break that habit). I'm not the maintainer though.

Thanks for your efforts here. I know it's devolved into a little bit of a bikeshed but you ended up bringing a lot of improvements which we've been waiting to do, so naturally, that brings a lot of discussion. Your next PR will hopefully not be so hard ;)

Both asturm and I are on IRC (#gentoo-proxy-maint on Freenode) if you find that easier to ask any quick questions by the way.

Reply:
Using upstream seems like a solid choice - I was just worried that it might be considered bad practice since GitHub probably has higher capacity than gitlab.freedesktop.org but, I guess, most users will get them from Gentoo mirrors anyway.

I hope so, too, but considering it will likely involve either getting sys-auth/rtkit to allow for higher RT priority by default (PW now prefers to run at 88) or getting the excellently named [hsphfpd](https://github.com/pali/hsphfpd-prototype), which PW can now use instead of oFono, into Portage tree, this may be the easier PR still.

Thank you for bringing that to my attention, I'll look into using IRC (probably via Matrix).